projects/storefrontlib/src/shared/components/truncate-text-popover/truncate.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+const defaultLimit = 20;
+
+@Pipe({
+  name: 'cxTruncate',
+})
+export class TruncatePipe implements PipeTransform {
+  /**
+   * example usage {{ exampleString | cxTruncate: [1, ''] }}
+   */
+  transform(
+    value: string,
+    args?: [number | undefined, (string | undefined)?]
```

Comment:
I think in first case it's ok to not allow `undefined`. There is no point for anyone just to pass empty array